### PR TITLE
Link to the internal committer permissions process

### PR DESCRIPTION
**Pull request by backes**
We often forget that there is a streamlined process for v8 team members. Link to the internal process from the external page to increase visibility.

### CODE REVIEW

**Comment by backes** [comment]
PTAL, especially about linking an internal document externally. Via go link, I think it's OK. Chromium also does that, e.g. at https://www.chromium.org/developers/how-tos/get-the-code.

File: src/docs/become-committer.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 title: 'Becoming a committer'
 description: 'How does one become a V8 committer? This document explains.'
 ---
+
+*Note to googlers: There is a [slightly different approach for v8 team members](http://go/v8/setup_permissions.md).*
```
Comment (mathiasbynens) [inline src/docs/become-committer.md]:
```suggestion
*Note to Googlers: There is a [slightly different approach for V8 team members](http://go/v8/setup_permissions.md).*
```

Comment (mathiasbynens) [inline src/docs/become-committer.md]:
```suggestion

*Note to Googlers: There is a [slightly different approach for V8 team members](http://go/v8/setup_permissions.md).*
```

**Comment by backes** [comment]
Thanks, done.

File: src/docs/become-committer.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 title: 'Becoming a committer'
 description: 'How does one become a V8 committer? This document explains.'
 ---
+
```
Comment (mathiasbynens) [inline src/docs/become-committer.md]: Nit: we don't usually have this newline here.

Comment (backes) [reply src/docs/become-committer.md]: Ack, removed.

**Pull request merged**